### PR TITLE
[8.x-1.x-exp-news] [Experimental Feature] News Content Type & documentation

### PR DESCRIPTION
**Pull request by dimb**
Issue https://github.com/ymcatwincities/openy/issues/415

Done features:
- News content types, similar to Blog but without Style options
- Paragraph to list all news
- Paragraph to show the latest news
- Featured News
- New Taxonomy vocabulary
- Paragraph to show the latest news by a branch
- Paragraph to show the latest news by a camp
- documentation for news

### CODE REVIEW

**Comment by gundevel** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by gundevel** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by gundevel** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

File: docs/Content structure/Paragraphs/Featured News Posts.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Featured News Posts
+This is a paragraph type that will be used for the featured stories.
```
Comment (ddrozdik) [inline docs/Content structure/Paragraphs/Featured News Posts.md]: `... for the featured news.` - please change last word

File: docs/Content structure/Taxonomy/News Category.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# News Category
+This is a vocabulary that will be used for adding blog categories on the site.
```
Comment (ddrozdik) [inline docs/Content structure/Taxonomy/News Category.md]: `for adding *news* categories on the site.`

**Review by ddrozdik — CHANGES_REQUESTED** [review]

**Comment by ddrozdik** [comment]
@dimb please fix comments ^^. Since this PR already include demo content, let's union both prs into one.

File: docs/Content structure/Content Types/News.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# News
+News Post content type is used for adding news posts on the site.
+
+### Fields
+| Name  | Machine name | Required | Description |
+| ------------- | ------------- | ------------- | ------------- |
+| Title  | drupal's default  | Yes | Title of the blog item. |
```
Comment (ddrozdik) [inline docs/Content structure/Content Types/News.md]: `Title of the blog item.` should be `Title of the news item.`

File: docs/Content structure/Paragraphs/Featured News Posts.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Featured News Posts
+This is a paragraph type that will be used for the featured news.
+
+### Fields
+| Name  | Machine name | Required | Description |
+| ------------- | ------------- | ------------- | ------------- |
+| Headline | field\_prgf_headline | No | Headline of the banner. |
+| Blog Posts | field\_fnews_posts | Yes | Multiple values. Reference field to News posts. |
```
Comment (ddrozdik) [inline docs/Content structure/Paragraphs/Featured News Posts.md]: `Blog Posts` should be `News Posts`

**Review by ddrozdik — CHANGES_REQUESTED** [review]

**Comment by ddrozdik** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2100 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2100 |
| Installation process | http://install.openy-dev.ffwua.com/build2100/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2100/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build2100

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2100/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2100/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2100/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2100/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2100/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build2100/profiles/contrib/openy/build/reports/behat

**Comment by ddrozdik** [comment]
@dimb we need to add to each views related to news "No Result behavior" and display this text `<h3 class="no-results">There is no news yet.</h3>`

**Review by ddrozdik — APPROVED** [review]

**Pull request merged**